utils/dev-tool/src/server.rs

```diff
@@ -1,80 +1,54 @@
-use crate::utils::{self, CommandRunner, HashInfo};
+use crate::utils::{self, CommandRunner, SERVER_PORT};
 use crate::ToolEnvironment;
 
-use rand::Rng;
 use std::process::{Command, Stdio};
 use std::time::Duration;
 use std::{fs, thread};
 
 pub fn run_server_detached(tool_env: &ToolEnvironment) {
     dotenv::from_path(utils::local_env_path(&tool_env.root_dir)).unwrap();
-    let mut port;
-    let server_db_dir = tool_env.server_dbs_dir.join(&tool_env.hash_info_dir);
 
     Command::new("cargo")
         .args(["build", "-p", "lockbook-server", "--release"])
         .assert_success();
 
-    loop {
-        port = rand::thread_rng().gen_range(1024..u16::MAX);
-
-        let mut run_result = Command::new(tool_env.target_dir.join("release/lockbook-server"))
-            .env("SERVER_PORT", port.to_string())
-            .env("INDEX_DB_LOCATION", &server_db_dir)
-            .current_dir(&tool_env.root_dir)
-            .stderr(Stdio::null())
-            .stdout(Stdio::null())
-            .spawn()
-            .unwrap();
+    let mut run_result = Command::new(tool_env.target_dir.join("release/lockbook-server"))
+        .env("SERVER_PORT", SERVER_PORT.to_string())
+        .env("INDEX_DB_LOCATION", &tool_env.server_db_dir)
+        .env("LOG_PATH", &tool_env.server_db_dir)
```

Comment:
Why are you overriding this at all? Why not use the default `test.env` values?

Reply:
Fair point, defaults can be used.